demo/src/plugins/dashboard/components/active-edits/index.jsx

```diff
@@ -0,0 +1,107 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { connect } from 'react-redux';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  Icon,
+  Table,
+} from '@triniti/admin-ui-plugin/components';
+import delegate from './delegate';
+import selector from './selector';
+import TableRow from './TableRow';
+
+class ActiveEdits extends React.Component {
+    static propTypes = {
+      accessToken: PropTypes.string.isRequired, // eslint-disable-line react/forbid-prop-types
+      collaborationNodes: PropTypes.array.isRequired, // eslint-disable-line react/forbid-prop-types
+      handleRequestCollaborationNodes: PropTypes.func.isRequired,
+      location: PropTypes.object.isRequired, // eslint-disable-line react/forbid-prop-types
+      title: PropTypes.oneOf(['Articles', 'Non-Article Nodes']).isRequired,
+    };
+
+    constructor(props) {
+      super(props);
+      this.handleRequestCollabNodes = this.handleRequestCollabNodes.bind(this);
+    }
+
+    componentDidMount() {
+      this.handleRequestCollabNodes();
+    }
+
+    componentDidUpdate(prevProps) {
+      const { location } = this.props;
+      const { location: prevLocation } = prevProps;
+      /**
+         * If user clicks same nav item they are already on (same pathname but different key)
+         * search again to refresh results.
+         */
+      if (
+        prevLocation.pathname === location.pathname
+            && prevLocation.key !== location.key
+      ) {
+        this.handleRequestCollabNodes();
+      }
+    }
+
+    handleRequestCollabNodes() {
+      const { handleRequestCollaborationNodes, accessToken } = this.props;
+      handleRequestCollaborationNodes(accessToken);
+    }
+
+    render() {
+      const { collaborationNodes, title } = this.props;
+      const nodeType = collaborationNodes.map((node) => node.schema().getCurie().getMessage());
```

Comment:
@nanetteCodes its cleaner to just separate the articles from other early instead of doing it multiple times below. filter the nodes into an array for articles and an array for other, then you can either map or show the not found message right away without filtering more

Reply:
refactored this